packages/headless/src/api/commerce/common/product.ts

```diff
@@ -83,6 +82,18 @@ export interface Product {
    * From the `ec_rating` field.
    */
   ec_rating?: number;
+  /**
+   * The gender the product is intended for.
+   */
+  ec_gender?: string;
+  /**
+   * The listing page the product was returned in.
+   */
+  ec_listing?: string;
```

Comment:
I don't think this is actually used anymore. 🤔 @anthonydelage ?

Reply:
I'll remove it... easier to add stuff than removing it once the sub-package is GA